pkg/utils/utils.go

```diff
@@ -225,3 +225,14 @@ func GetObjectTypeMetaData(obj interface{}) metaV1.TypeMeta {
 	}
 	return typeMeta
 }
+
+// DeleteDoubleWhiteSpace returns slice that removing whitespace from a arg slice
+func DeleteDoubleWhiteSpace(slice []string) []string {
+	result := []string{}
+	for _, s := range slice {
+		if s != " " {
```

Comment:
@nnao45 This doesn't work. You need to check if `s` empty
```
@@ -230,7 +230,7 @@ func GetObjectTypeMetaData(obj interface{}) metaV1.TypeMeta {
 func DeleteDoubleWhiteSpace(slice []string) []string {
        result := []string{}
        for _, s := range slice {
-               if s != " " {
+               if len(s) != 0 {
                        result = append(result, s)
                }
        }
```
This worked for me. Can you please fix this?

Reply:
sorry, i fix  this